devices/Ld2410/Ld2410/Extensions/EndiannessExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Ld2410.Extensions
+{
+    internal static class EndiannessExtensions
```

Comment:
There is the https://github.com/nanoframework/nanoFramework.IoT.Device/tree/develop/devices/System.Buffers.Binary.BinaryPrimitives is converting anything to Little Endian. It may be better than looking at the BitConverter

Reply:
Ahh very cool. I didn't know about this class. Will try to migrate as much as possible to using it. Thanks :)